manager/src/main/java/com/redhat/service/bridge/manager/connectors/creation/CreateConnectorWorker.java

```diff
@@ -0,0 +1,99 @@
+package com.redhat.service.bridge.manager.connectors.creation;
+
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.openshift.cloud.api.connector.models.Connector;
+import com.openshift.cloud.api.connector.models.ConnectorRequest;
+import com.openshift.cloud.api.connector.models.DeploymentLocation;
+import com.openshift.cloud.api.connector.models.KafkaConnectionSettings;
+import com.openshift.cloud.api.connector.models.ServiceAccount;
+import com.redhat.service.bridge.infra.models.dto.ConnectorStatus;
+import com.redhat.service.bridge.manager.connectors.AbstractConnectorWorker;
+import com.redhat.service.bridge.manager.connectors.ConnectorsApiClient;
+import com.redhat.service.bridge.manager.models.ConnectorEntity;
+
+import io.quarkus.vertx.ConsumeEvent;
+
+import static com.redhat.service.bridge.manager.connectors.Events.KAFKA_TOPIC_CREATED_EVENT;
+
+@ApplicationScoped
+public class CreateConnectorWorker extends AbstractConnectorWorker<Connector> {
+    public static final String KAFKA_ID_IGNORED = "kafkaId-ignored";
+
+    @ConfigProperty(name = "managed-connectors.cluster.id")
+    String mcClusterId;
+
+    @ConfigProperty(name = "kafka.bootstrap.servers")
+    String kafkaBootstrapServer;
+
+    @ConfigProperty(name = "managed-connectors.kafka.client.id")
+    String serviceAccountId;
+
+    @ConfigProperty(name = "managed-connectors.kafka.client.secret")
+    String serviceAccountSecret;
+
+    @Inject
+    ConnectorsApiClient connectorsApiClient;
+
+    @ConsumeEvent(value = KAFKA_TOPIC_CREATED_EVENT, blocking = true)
```

Comment:
In the other workers you use `Events.CONSTANT` and here you use static import (also in other workers), maybe one single approach would improve readability